starlark/library.go

```diff
@@ -783,6 +783,9 @@ func ord(thread *Thread, _ *Builtin, args Tuple, kwargs []Tuple) (Value, error)
 
 // https://github.com/google/starlark-go/blob/master/doc/spec.md#print
 func print(thread *Thread, fn *Builtin, args Tuple, kwargs []Tuple) (Value, error) {
+	if len(kwargs) > 0 {
+		return nil, fmt.Errorf("print does not accept keyword arguments")
+	}
```

Comment:
Please delete the code at 801--811. You can them eliminate sep and change the first loop to:

  for i, v := ... {
    if i > 0 { buf.WriteString(", ") }
    ...
  }

I have no idea why GitHub thinks I shouldn't be able to comment on lines not directly modified by a change.